utils/gyb_syntax_support/protocolsMap.py

```diff
@@ -14,6 +20,15 @@
     'MemberDeclList': [
         'MemberDeclBlock'
     ],
+    'FunctionCallExpr': [
+        'CodeBlockItem',
```

Comment:
For a future PR: Would it make sense to add automatically conformances like `ExpressibleAsFunctionCallExpr : ExpressibleAsExpr` (just like we have `FunctionCallExpr : ExprBuildable` and then add `ExpressibleAsExpr : ExpressibleAsCodeBlockItem`? That way, we wouldn’t need to add conformances to `ExpressibleAsCodeBlockItem` for every expression node.

The same thing I just showed for expression also applies to statements.

Reply:
Good idea!